### DISCUSSION TITLE
I was about to give up on Postgres

### DISCUSSION THREAD

**Discussion by katenecon58** [dbeaver/dbeaver#11094] [Q&A] [answered]
Turns out I just had to switch away from pgAdmin. It's a shame that it's the first result when you search "postgres gui windows".

2 minutes into DBeaver (which I've actually used for MSSQL but didn't know worked with PSQL) and much happier.

**Comment by p91paul** [comment, marked as answer]
Next time you start on a new database, just assume dbeaver supports it 😆 they pretty rightfully advertise it as the **universal** database client!
